### DISCUSSION TITLE
Possible to add support for Sylcin (Seplos clone) BMSs?

### DISCUSSION THREAD

**Discussion by no1knows** [syssi/esphome-seplos-bms#66] [Ideas]
Hello, I have three Taico batteries that use a Sylcin BMS (clones of Seplos). Lots of discussion about these batteries here: https://www.photovoltaikforum.com/thread/167402-erfahrungen-mit-taico-lifepo4-48v-speicher-packs/?postID=3252835#post3252835

I wonder if it'd be possible to add support for these BMSs to esphome-seplos-bms? I know Taico batteries have become popular, especially in Germany. 

In case useful, support for Sylcin has been added to the BSC project: https://github.com/shining-man/bsc_fw/blob/main/src/devices/SylcinBms.cpp. I'm hoping it can be easily ported, but don't have the skills to do it myself. 

Thank you!

**Comment by syssi** [comment]
I will have a look.

Payload examples for further analysis: https://github.com/shining-man/bsc_fw/blob/main/src/devices/SylcinBms.cpp#L115-L118
https://github.com/shining-man/bsc_fw/blob/main/src/devices/SylcinBms.cpp#L310-L324
